docs/references.bib

```diff
@@ -63,3 +63,22 @@ @article{SLH70
     year = {1970},
     doi = {10.1021/i160034a013}
 }
+
+
+
+@article{Lor63,
+  title={Deterministic nonperiodic flow},
+  author={Lorenz, Edward N},
+  journal={Journal of atmospheric sciences},
+  volume={20},
+  number={2},
+  pages={130--141},
+  year={1963}
+}
+
+@book{Str18,
+  title={Nonlinear dynamics and chaos with student solutions manual: With applications to physics, biology, chemistry, and engineering},
+  author={Strogatz, Steven H},
+  year={2018},
+  publisher={CRC press}
```

Comment:
Include doi, eddition, and ISBN: [10.1201/9780429492563](https://doi.org/10.1201/9780429492563)